specification/cognitiveservices/AnomalyDetector/multivariate/models.tsp

```diff
@@ -81,32 +79,22 @@ model MultivariateBatchDetectionResultSummary {
   errors?: ErrorResponse[];
 
   @doc("Variable status.")
-  variableStates?: VariableState[];
+  variableStates?: MultivariateVariableState[];
 
   @doc("""
 Detection request for batch inference. This is an asynchronous inference that
 will need another API to get detection results.
 """)
-  setupInfo: MultivariateBatchDetectionOptions;
-}
-
-@doc("Error information that the API returned.")
-@error
-model ErrorResponse {
-  @doc("Error code.")
-  code: string;
-
-  @doc("Message that explains the error that the service reported.")
-  message: string;
+  setupInfo: MultivariateBatchDetectionSettings;
 }
 
 @doc("Variable status.")
-model VariableState {
+model MultivariateVariableState {
   @doc("Variable name in variable states.")
-  variable?: string;
+  variable: string;
```

Comment:
This is a change to REST API definition -- `variable` changes from optional to required. Is this intentional?

This is safe if the model is "output only" -- is that the case?

Reply:
Yes this change is intentional and yes the model is "output only".